lib/embed/stacks_media_stream.rb

```diff
@@ -47,7 +47,8 @@ def streaming_file_prefix
       if video?
         'mp4'
       elsif audio?
-        'mp3'
+        return 'mp3' if file_extension == 'mp3'
+        return 'mp4' if file_extension == 'm4a'
```

Comment:
This feels a bit unsatisfactory to me.  The entire method was predicated on the fact that audio always is `mp3` and video is always `mp4`.  I wonder if should rethink this method entirely now.  I don't have a good idea off the top of my head what that might look like, but perhaps a config hash that can have a default file prefix and then can configure variations from that default (e.g. `m4a` gets `mp4` instead of the default `mp3`).

That being said, this is a small enough change that if it felt like setting that up was more trouble that it was worth, we can wait until we come across the next incompatible file prefix to figure it out.  Totally up to you.